Repo: Build failure on "Cannot find name 'util'"

### Suggestion

Introduced because I merged #7708, which added in another usage of `util`, without merging in the changes from #7669.

```plaintext
  ➤ YN0000: │ @typescript-eslint/typescript-eslint@workspace:. STDOUT > nx run eslint-plugin:build
  Error: ➤ YN0000: │ @typescript-eslint/typescript-eslint@workspace:. STDOUT src/rules/no-unsafe-return.ts(109,13): error TS2304: Cannot find name 'util'.
  ➤ YN0000: │ @typescript-eslint/typescript-eslint@workspace:. STDOUT  
  ➤ YN0000: │ @typescript-eslint/typescript-eslint@workspace:. STDOUT  >  NX   Running target build for 13 projects and 1 task they depend on failed
```

--- a/packages/eslint-plugin/src/rules/no-unsafe-return.ts
+++ b/packages/eslint-plugin/src/rules/no-unsafe-return.ts
@@ -12,6 +12,7 @@ import {
   getThisExpression,
   isAnyOrAnyArrayTypeDiscriminated,
   isTypeAnyType,
+  isTypeFlagSet,
   isTypeUnknownArrayType,
   isTypeUnknownType,
   isUnsafeAssignment,
@@ -106,7 +107,7 @@ export default createRule({
         for (const signature of functionType.getCallSignatures()) {
           if (
             returnNodeType === signature.getReturnType() ||
-            util.isTypeFlagSet(
+            isTypeFlagSet(
               signature.getReturnType(),
               ts.TypeFlags.Any | ts.TypeFlags.Unknown,
             )

